Change dot-atom keymap.cson

isoeugenol opened the pull request:
- ctrl-shift-p and ctrl-p exist only macOS keymap
- i think example is same all platforms

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change

<!--

We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.

-->

### Alternate Designs

<!-- Explain what other alternates were considered and why the proposed version was selected -->

### Why Should This Be In Core?

<!-- Explain why this functionality should be in atom/atom as opposed to a package -->
dot-atom keymap.cson example is only exist macOS 
i think example should work all platforms.
### Benefits

<!-- What benefits will be realized by the code change? -->
if someone use example keymaps they work all platfroms
### Possible Drawbacks

<!-- What are the possible side-effects or negative impacts of the code change? -->

### Applicable Issues

<!-- Enter any applicable Issues here -->

lee-dohm commented:
Thanks for the submission!

The point of the examples is to simply give things that could be mapped if one chose to and to give people a basic understanding of the syntax. They aren't intended to be actual keybindings that one would want to keep, only to experiment with. Also, since not all keyboards have a <kbd>PgUp</kbd> or <kbd>PgDn</kbd> key (most notably many Mac keyboards don't) but every modern keyboard has a <kbd>Ctrl</kbd>, <kbd>Shift</kbd>, and <kbd>P</kbd> key, I feel the current examples work well for their intended purpose.

Thanks again for helping out! I'm going to close this but if you have any questions, please let us know :+1:

isoeugenol commented:
I have never used a Mac, so I can not imagine that reason.
when i was using window and linux i tried the example. but it is don`t work.
How do you feel about changing to "shift-up, shift-down"?

lee-dohm commented:
They're just examples to show the format, not something that people would seriously want in their keymap. For that goal, the current examples are fine. Thanks for your help.

isoeugenol committed:
Change dot-atom keymap.cson

- ctrl-shift-p and ctrl-p exist only macOS keymap
- i think example is same all platforms
